app/config/AppConfig.scala

```diff
@@ -46,6 +46,8 @@ class FrontendAppConfig @Inject()(servicesConfig: ServicesConfig) extends AppCon
   lazy val personalIncomeTaxSubmissionUrl: String =s"$personalIncomeTaxSubmissionBaseUrl/update-and-submit-income-tax-return/personal-income"
   lazy val additionalInformationSubmissionBaseUrl: String = servicesConfig.getString(ConfigKeys.additionalInformationFrontendUrl)
   lazy val additionalInformationSubmissionUrl: String =s"$additionalInformationSubmissionBaseUrl/update-and-submit-income-tax-return/additional-information"
+  lazy val tailorReturnBaseUrl: String = servicesConfig.getString(ConfigKeys.tailorReturnFrontendUrl)
```

Comment:
As there is existing tailoring journey, may be we need to name these variables accordingly. will chat and decide

Reply:
also i feel may be we should not use lazy, otherwise we will not know if any config is missing before deployment.